homeassistant/components/palazzetti/const.py

```diff
@@ -0,0 +1,18 @@
+"""Constants for the Palazzetti integration."""
+
+from datetime import timedelta
+import logging
+from typing import Final
+
+DOMAIN: Final = "palazzetti"
+PALAZZETTI: Final = "Palazzetti"
+LOGGER = logging.getLogger(__package__)
+SCAN_INTERVAL = timedelta(seconds=5)
```

Comment:
I find this to be a very low scan interval. I don't believe that a heater changes every 5 seconds. If you want the device to be responsive after an action has been done, be sure to call `self.coordinator.async_refresh()` to get a new state